repos/system_upgrade/common/libraries/rhsm.py

```diff
@@ -383,6 +384,44 @@ def switch_certificate(context, rhsm_info, cert_path):
             context.copy_to(cert_path, os.path.join(path, os.path.basename(cert_path)))
 
 
+def is_rhsm_registered(context):
+    """
+    Check whether the system is registered with Red Hat Subscription Manager
```

Comment:
@matejmatuska @ptoscano based on the content, it seems that function is not testing whether the system is registered, but whether it's actually subscribed correctly.

Reply:
hm no, that should be ok:
- `sub-man identity` -> exit code = 0 -> registered
- `sub-man identity` -> exit code = 1 -> not registered
- everything else -> error

Reply:
hmm.. oki. so I am confused just by the error in the end. Discussed that with @matejmatuska  I think I understand it now